azimuth/utils/ml/model_performance.py

```diff
@@ -24,3 +25,27 @@ def sorted_by_utterance_count_with_last(metrics: List[T], index_to_put_first: in
     """
     first = metrics.pop(index_to_put_first)
     return [first, *sorted_by_utterance_count(metrics)]
+
+
+def compute_outcome(prediction: int, label: int, rejection_class_idx) -> OutcomeName:
+    """Compute prediction outcome based on the prediction and label.
+
+    Note: It was moved out of the OutcomesModule class for circular import issues.
+
+    Args:
+        prediction: Class index of the prediction
+        label: Class index of the label
+        rejection_class_idx: Class index of the rejection class
+
+    Returns:
+        Outcome value
+    """
+    if prediction == label:
+        if label == rejection_class_idx:
+            return OutcomeName.CorrectAndRejected
+        else:
+            return OutcomeName.CorrectAndPredicted
+    elif prediction == rejection_class_idx:
+        return OutcomeName.IncorrectAndRejected
+    else:
+        return OutcomeName.IncorrectAndPredicted
```

Comment:
I know this is not new code, but I find it hard to read, considering how simple it is conceptually. I thought I'd give a small suggestion to bring a little bit more structure.
```suggestion
    rejected = prediction == rejection_class_idx
    if prediction == label:
        return OutcomeName.CorrectAndRejected if rejected else OutcomeName.CorrectAndPredicted
    else:
        return OutcomeName.IncorrectAndRejected if rejected else OutcomeName.IncorrectAndPredicted
```